packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/batch/submit-job.ts

```diff
@@ -337,4 +346,24 @@ export class BatchSubmitJob extends sfn.TaskStateBase {
       ResourceRequirements: resources.length ? resources : undefined,
     };
   }
+
+  /**
+   * Validate the specified tags according to
```

Comment:
I think the function is very well written and self explanatory. Do we really need the docstring? What happens if the URL will change in the future?